include/boost/geometry/algorithms/detail/buffer/get_piece_turns.hpp

```diff
@@ -288,10 +287,8 @@ class piece_turn_visitor
 
     template <typename Section>
     inline bool apply(Section const& section1, Section const& section2,
-                    bool first = true)
+                    bool = true)
```

Comment:
So we can remove it. I'm fine with it like this in this PR, we can change it in a follow up